### DISCUSSION TITLE
Vote for a new Wrapper

### DISCUSSION THREAD

**Discussion by kiwicopple** [supabase/supabase#26627] [Feature Requests]
This issue is for tracking requests/demand for Wrappers. Give a thumbs up if you want an extension added.

Please refrain from adding comments unless you are adding an extension which isn't already listed.

**Comment by michaeldever** [comment]
Would love to see support for Paddle as part of this - www.paddle.com for context. Love this either way, even if paddle never makes it in. 

Great job supateam!

**Comment by burmecia** [comment]
It will be great if we can have a FDW for Auth0 or Cognito.

Developers migrating from Auth0 or Cognito require some way to export user or database information and backfill that into Supabase Auth. Or, with Multi-JWT support, they need a way to cross-reference user identifiers from those systems with the JWT issued by those systems. Foreign data wrappers would be incredibly useful when migrating to Supabase Auth.

**Comment by burmecia** [comment]
Snowflake FDW will be a good choice.

**Reply by Slyracoon23** [reply]
Definitely need snowflake - we are using it with LLMs and analytics. @burmecia

**Comment by Subsegment** [comment]
[CnosDB](https://www.cnosdb.com)(https://www.cnosdb.com/) FDW would be also a good option as it can replace TimescaleDB or some other time-series database. Here is the discussion thread: [https://github.com/supabase/wrappers/discussions/187](https://github.com/supabase/wrappers/discussions/187) .

**Reply by cnoshb** [reply]
This is a good one.

**Comment by elvise** [comment]
What about FDW oracle DB ? 🙂

**Comment by jacobwilson41** [comment]
Would love to see support for Airtable's linked records field type in the FDW

**Comment by asselinpaul** [comment]
Clerk would be great (https://github.com/tembo-io/clerk_fdw)

**Reply by TheOtherBrian1** [reply]
A long time ago, I created a [pseudo FDW](https://gist.github.com/TheOtherBrian1/7d1b8b931d66c8b5f783f536a4a5bf97) for Clerk

**Comment by BhaskaranR** [comment]
FDW for Amazon QLDB would be great. please consider.

**Comment by DuncanLHS** [comment]
Microsoft Dataverse (PowerApps/Dynamics 365)

**Comment by burmecia** [comment]
Salesforce FDW (https://developer.salesforce.com/docs/apis) would be nice one to have.

**Comment by jachinte** [comment]
OpenFGA FDW (https://openfga.dev/) -- this would be nice to manage authorization using ReBAC.

**Comment by riderx** [comment]
Cloudflare D1

**Comment by burmecia** [comment]
MongoDB

**Comment by burmecia** [comment]
DynamoDB

**Comment by riderx** [comment]
Cloudflare Workers analytics (they own custom and cheap clickhouse):
https://developers.cloudflare.com/analytics/analytics-engine/sql-api/

**Comment by burmecia** [comment]
Firebase Data Connect: https://firebase.google.com/docs/data-connect

**Comment by MartinJrP** [comment]
CloudKit (iCloud): https://developer.apple.com/icloud/cloudkit/

**Comment by picsoung** [comment]
Typeform https://www.typeform.com/developers/responses/

**Comment by burmecia** [comment]
MySQL: https://www.mysql.com/

**Comment by rebwar-bajallan-odevo** [comment]
Microsoft Fabric Warehouses: https://learn.microsoft.com/en-us/fabric/data-warehouse/data-warehousing
